### PR TITLE
config changes

### PR DESCRIPTION
**Pull request by jenneron**

### CODE REVIEW

File: kernel/configs/pmos.config [hunk 1]
```diff
@@ -53,7 +53,7 @@ CONFIG_ASHMEM=y
 CONFIG_ANDROID=y
```
Comment (maximschwalm) [inline kernel/configs/pmos.config:53]: Should we remove this deprecated option and enable PM_USERSPACE_AUTOSLEEP instead?

Comment (digetx) [reply kernel/configs/pmos.config:53]: We still have CONFIG_ANDROID in grate kernel for legacy ramconsole, but we don't really need that kconfig option and may remove it.

PM_USERSPACE_AUTOSLEEP should make system unusable, it's intended for Android only.

Comment (maximschwalm) [reply kernel/configs/pmos.config:53]: It's no longer possible to build legacy ramconsole.

Comment (digetx) [reply kernel/configs/pmos.config:53]: Indeed, I missed that the Kconfig needed more changes after last rebase.

Comment (digetx) [reply kernel/configs/pmos.config:53]: I'll fix the ramconsole on the next rebase.

**Comment by digetx** [comment]
Thanks

**Pull request merged**